### ISSUE TITLE
Drive button add-on error

### ISSUE THREAD

**Issue by skintfr** [tablacus/TablacusExplorerAddons#28]
Hi,
When clicking on drive button add-on icon, i have the following error 
Line: 32  
Erreur : Impossible d’obtenir la propriété  « length » d’une référence null ou non définie
which can be translated by something like  :  "Impossible to recover the propriety 'lenght' of a null reference or not definied"

Any known issue or solution ?

Thanks. For the rest this is a very great job.

**Comment by tablacus** [comment]
I haven't encountered this.
Anyway, I adjusted this with an error message.
Please update an add-on of  "Drive button".

Best regards,

**Comment by skintfr** [comment]
Hi,
Perfect, works very well.
Thanks a lot.

Just another issue but don't know if this is not due to our network security policy at my office. When clicking on "Tools/Add-ons/get Add-ons" i have the error : 
 Network error (403)
http://tablacus.github.io/TablacusExplorerAddons/index.xml?25379387

I can display the content of the xml file when typing the URL in my browser.
No issue to get Add-ons view on the same PC yesterday when i was connected on my home WIFI.

Thanks a lot and great job.

**Comment by skintfr** [comment]
Hi,

Inetrent Explorer is blocked by a security policy, we can't use it to go on internet. Default browser is Chrome and it works fine wih it.
URL works also with my "default" browser Firefox.

Best Regards

**Comment by tablacus** [comment]
It may be a cause.
Tablacus Explorer is using Internet Explorer components.

Thank you,

**Comment by skintfr** [comment]
Hi,

No way to workaround ?

Thank you

**Comment by tablacus** [comment]
Would you execute this sctipt in Tablacus Explorer?

> var FV = GetFolderView(Ctrl, pt);
> api.URLDownloadToFile(null, "http://tablacus.github.io/TablacusExplorerAddons/index.xml", fso.BuildPath(FV.FolderItem.Path, "index.xml"));
 
![image](https://user-images.githubusercontent.com/5156977/38302880-31e50258-383f-11e8-8d7e-67d41ecc41b2.png)

Download http://tablacus.github.io/TablacusExplorerAddons/index.xml to current folder.

Click "Download"
![image](https://user-images.githubusercontent.com/5156977/38302998-a00e1710-383f-11e8-9e1b-13fd8dbbb705.png)

Downloaded.
![image](https://user-images.githubusercontent.com/5156977/38303046-cdef9ed8-383f-11e8-80b4-8a46fd9fa252.png)


Best regards,

**Comment by skintfr** [comment]
hi,
I have tried to follow your guide but i probably missed something 👍 
![image](https://user-images.githubusercontent.com/16616075/38306342-8cd79c2c-3810-11e8-918e-2a7598e1ff38.png)

**Comment by tablacus** [comment]
Ok, please push "Test"
![image](https://user-images.githubusercontent.com/5156977/38306671-4bfba7de-384c-11e8-9236-7c1dac80d0aa.png)

Best regards,

**Comment by skintfr** [comment]
Nothing happens when clicking on Test button.
FYI, I have removed my proxy settings on IE11 and i can access your site now but it changed nothing for my problem.

**Comment by tablacus** [comment]
Ok, I will consider another way.

Thank you,

**Comment by tablacus** [comment]
How about bitadmin.exe?

Command line.

> bitsadmin.exe /TRANSFER dl https://tablacus.github.io/TablacusExplorerAddons/index.xml Q:\USB_Apps\Accessoires\Tablacus_Explorer

![image](https://user-images.githubusercontent.com/5156977/38308595-966d6b94-3852-11e8-8ce4-c7e429d5fe0a.png)

Thank you,

**Comment by skintfr** [comment]
Tried 3 times, sorry

**Comment by tablacus** [comment]
Thank you for information.
I will consider a way to workaround.

Thank you very much,

**Comment by tablacus** [comment]
I've made an add-on "Download".
Please try to update Tablacus Explorer to latest version and install an add-on "Download".
(Extract zip file to addons folder)

This add-on uses "PowerShell", "cURL", "Wget" or "BITSADMIN" instead of  Internet Explorer components.
To use "cURL" or "Wget", set the full path of the exe file.

![image](https://user-images.githubusercontent.com/5156977/38373017-22b6eab6-392b-11e8-9194-5749e6363afd.png)

Best regards,

**Comment by tablacus** [comment]
My pleasure,
